### PR TITLE
re-arange topics into directories for easier navigation

### PR DESCRIPTION
**Pull request by lauranovich**
fixes organization issues 
created directory structure and index files

### CODE REVIEW

**Comment by amnonh** [comment]
@tzach please review

**Comment by lauranovich** [comment]
before
Screenshot from 2021-04-01 16-50-15
after
Screenshot from 2021-04-01 16-49-40

**Comment by tzach** [comment]
build warnings
```
/docs/source/dashboards/index.rst:5: WARNING: toctree contains reference to nonexisting document 'dashboards/advisor/index'
/docs/source/dashboards/index.rst:17: WARNING: unknown document: advisor/index
```

File: docs/source/dashboards/index.rst [hunk 1]
```diff
@@ -0,0 +1,17 @@
+======================================
+Grafana Dashboards Configuration Guide
```
Comment (tzach) [inline docs/source/dashboards/index.rst]: This is not following the tree in https://github.com/scylladb/scylla-monitoring/issues/1346

File: docs/source/dashboards/index.rst [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+   Adding and Modifying Dashboards <updating_dashboard>
+   CQL Optimization Dashboard <cql_optimization>
+   Scylla Monitoring Stack Advisor <advisor/index>
+
+There are several dashboards you can use with Scylla Monitoring Stack. Choose a topic to begin:
+
+* :doc:`Adding and Modifying Dashboards <updating_dashboard>`
+* :doc:`CQL Optimization Dashboard <cql_optimization>`
+* :doc:`Scylla Monitoring Stack Advisor <advisor/index>`
```
Comment (tzach) [inline docs/source/dashboards/index.rst]: link does not work

File: docs/source/dashboards/index.rst [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+   :hidden:
+
+   Adding and Modifying Dashboards <updating_dashboard>
+   CQL Optimization Dashboard <cql_optimization>
+   Scylla Monitoring Stack Advisor <advisor/index>
+
+There are several dashboards you can use with Scylla Monitoring Stack. Choose a topic to begin:
+
+* :doc:`Adding and Modifying Dashboards <updating_dashboard>`
+* :doc:`CQL Optimization Dashboard <cql_optimization>`
```
Comment (tzach) [inline docs/source/dashboards/index.rst]: Why is CQL Optimization Dashboard under "Grafana Dashboards Configuration Guide"?

**Review by tzach — COMMENTED** [review]
High level comments:
- The tree is different than the proposal at https://github.com/scylladb/scylla-monitoring/issues/1346

- No redirections (maybe not as bad)

File: docs/source/dashboards/index.rst (hunk 1 above)
Comment (lauranovich) [reply docs/source/dashboards/index.rst]: I didn't see that issue when I started this PR. I'll make the changes accordingly.

Comment (lauranovich) [reply docs/source/dashboards/index.rst]: once we're happy with the new organization, I'll make the redirections. We currently don't have a redirections file so I'll have to add one.

**Comment by tzach** [comment]
Much better.
High level comment: we do not need "Scylla Monitoring Stack" on each header and each page, its generate a lot of noise.
All pages is already under Scylla Monitoring Stack

Example:
Troubleshooting Guide for Scylla Monitoring Stack --> Troubleshooting
Download and Install Scylla Monitoring Stack --> Download and Install
Scylla Monitoring Stack Support Matrix --> Support Matrix

etc

**Comment by tzach** [comment]
LGTM
No need for redirect if it part of a future release (e.g. 3.7)

**Review by tzach — APPROVED** [review]

**Comment by amnonh** [comment]
@larisau, @tzach is this suppose to close #1346 ?

**Pull request merged**